### PR TITLE
updated gitpod yml

### LINKED ISSUE #79: docker-compose freezing after building successfully
**Describe the bug**
After building successfully, `docker-compose` freezes on the following section:
```
(18:29:37) INFO: Analyzed target //:manage (0 packages loaded, 0 targets configured).
(18:29:37) INFO: Found 1 target...
Target //:manage up-to-date:
  bazel-bin/manage
  bazel-bin/manage.zip
(18:29:37) INFO: Elapsed time: 1.356s, Critical Path: 0.01s
(18:29:37) INFO: 1 process: 1 internal.
(18:29:37) INFO: Build completed successfully, 1 total action
(18:29:37) INFO: Build completed successfully, 1 total action
django-api  | 2021-01-25 18:29:42,488 - Logger - INFO - Starting covidX app
django-api  | 2021-01-25 18:29:42,489 - Logger - INFO - BASE_DIR= /tmp/Bazel.runfiles_zcnxqi2m/runfiles/covidx
django-api  | System check identified some issues:
django-api  |
django-api  | WARNINGS:
django-api  | ?: (urls.W001) Your URL pattern '^apihealth/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  | ?: (urls.W001) Your URL pattern '^login-rest-router/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  | ?: (urls.W001) Your URL pattern '^rest-auth/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  | 2021-01-25 18:29:46,783 - Logger - INFO - Starting covidX app
django-api  | 2021-01-25 18:29:46,783 - Logger - INFO - BASE_DIR= /tmp/Bazel.runfiles_zcnxqi2m/runfiles/covidx
django-api  | System check identified some issues:
django-api  |
django-api  | WARNINGS:
django-api  | ?: (urls.W001) Your URL pattern '^apihealth/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  | ?: (urls.W001) Your URL pattern '^login-rest-router/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  | ?: (urls.W001) Your URL pattern '^rest-auth/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  |
django-api  | System check identified 3 issues (0 silenced).

```

**To Reproduce**
1. Clone the repository
2. Add a `.env` file in the cloned repository.
3. Run `docker-compose up --build` 
4. Wait for building to complete
5. See error

**Expected behavior**
Build completed successfully and command terminates

**Screenshots**
The build process freezes upon reaching this section:
image

**Desktop:**
 - OS: Windows 10

### LINKED ISSUE #94: cache bazel build
**Is your feature request related to a problem? Please describe.**

`docker-compose up --build` takes a long time every time due to `bazel` downloading plenty of dependencies that are normally cached locally. The cache directory inside the docker container is not persisted so the download happens every time.

**Describe the solution you'd like**

Add a bazel cache to the docker-compose file so that at least the cache would reside on a volume until `docker-compse down` is issued.

**Describe alternatives you've considered**

None

### LINKED ISSUE #108: Timeout when fetching @my_deps
**Describe the bug**
After running `docker-compose up --build`, a timeout occurs when fetching `@my_deps`.

**To Reproduce**
1. Clone the repository
2. Add a `.env` file in the cloned repository.
3. Run `docker-compose up --build` 
4. Wait for building to reach stage of timing out while fetching `@my_deps`
5. See error

**Expected behavior**
`@my_deps` fetched successfully without timeout

**Desktop:**
 - OS: Ubuntu 20.04.2.0 LTS

### PR DESCRIPTION
**Pull request by codecakes**
closes #108 fixes #94 closes #79 #78 closes #109

<a href="https://gitpod.io/#https://github.com/Xcov19/covidX/pull/122"></a>

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/covidx/122)

### CODE REVIEW

**Review by milan-tom — CHANGES_REQUESTED** [review]
Gitpod build fails with following message:
```
Step 10/31 : RUN apt install snapd
 ---> Running in b27d23db39ce

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package snapd


Error: build failed: cannot build base image: The command '/bin/sh -c apt install snapd' returned a non-zero code: 100
```

**Comment by milan-tom** [comment]
The change in 953cb88 to the `WORKSPACE` file has already been implemented in #109.

File: WORKSPACE [hunk 1]
```diff
@@ -3,13 +3,13 @@ workspace(name = "covidx")
...
-    urls = ["https://github.com/ali5h/rules_pip/archive/3.0.0.tar.gz"],
-)
-load("@com_github_ali5h_rules_pip//:defs.bzl", "pip_import")
+#http_archive(
+#    name = "com_github_ali5h_rules_pip",
+#    strip_prefix = "rules_pip-3.0.0",
+#    sha256 = "630a7cab43a87927353efca116d20201df88fb443962bf01c7383245c7f3a623",
+#    urls = ["https://github.com/ali5h/rules_pip/archive/3.0.0.tar.gz"],
+#)
+#load("@com_github_ali5h_rules_pip//:defs.bzl", "pip_import")
```
Comment (milan-tom) [inline WORKSPACE]: This change has already been implemented in #109.

**Comment by codecakes** [comment]
closes #108 fixes #94  closes #79 #78 closes #109

**Review by codeweaver22 — APPROVED** [review]

**Comment by milan-tom** [comment]
Closed #109

**Pull request merged**